webviz_subsurface/containers/_history_match.py

```diff
@@ -99,7 +97,6 @@ def _iterations_to_dict(iterations, labels):
     @property
     def layout(self):
         return html.Div([
-                  html.H2(self.title),
                   wsc.HistoryMatch(id=self.hm_id, data=self.hm_data)
                ])
```

Comment:
Can this now be changed to
`return wsc.HistoryMatch(id=self.hm_id, data=self.hm_data)`
?

Reply:
They could, but I think having them wrapped in a div is something we will need later on. To control styling and sizing and such. I think we can just keep the div's there for now.